images/win/windows2019.json

```diff
@@ -326,6 +325,16 @@
                 "{{ template_dir }}/scripts/Installers/Install-CodeQLBundle.ps1"
             ]
         },
+        {
+            "type": "powershell",
+            "environment_vars": [
+                "TOOLSET_JSON_PATH={{user `toolset_json_path`}}"
+            ],
+            "scripts": [
```

Comment:
Please avoid creating of new section. Use the existing one. Please put these scripts where `Install-YAMLLint.ps1` was run previously

Reply:
Moved it